blocks/article-feed/article-feed.js

```diff
@@ -1,17 +1,3 @@
-/*
- * Copyright 2021 Adobe. All rights reserved.
- * This file is licensed to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License. You may obtain a copy
- * of the License at http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under
- * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- * OF ANY KIND, either express or implied. See the License for the specific language
- * governing permissions and limitations under the License.
- */
-/* global */
-/* eslint-disable import/named, import/extensions */
-
```

Comment:
Why no license ?

Reply:
That was part of a discussion I once had with @davidnuescheler when we talked about switching to plain `airbnb-base` instead of `@adobe/eslint-helix` for projects: we found that mandating the license header in each and every block file that is requested by the browser separately is adding an unnecessary traffic overhead.

I did keep the license in `/scripts/scripts.js` as the starting point for all JS in the project.